Makefile

```diff
@@ -41,27 +47,37 @@ test:
 	done
 .PHONY: test
 
-prep-for-run:
+copy-k8s-sa:
+	mkdir -p ${TLS_CERTS_BASEDIR} || true
+	oc -n ${NAMESPACE} get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name} > _output/espod && \
+	oc -n ${NAMESPACE} exec -c elasticsearch $$(cat _output/espod) -- cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt > _output/ca.crt && \
+	oc -n ${NAMESPACE} serviceaccounts get-token elasticsearch > _output/sa-token && \
+	echo ${NAMESPACE} > _output/namespace && \
+	sudo mkdir -p /var/run/secrets/kubernetes.io/serviceaccount/||:  && \
+	sudo ln -sf $${PWD}/_output/ca.crt /var/run/secrets/kubernetes.io/serviceaccount/ca.crt && \
+	sudo ln -sf $${PWD}/_output/sa-token /var/run/secrets/kubernetes.io/serviceaccount/token
+.PHONY: copy-k8s-sa
+
+copy-es-certs:
+	mkdir -p ${TLS_CERTS_BASEDIR} || true
+ifneq ($(ES_CERTS_DIR), "")
+	cp ${ES_CERTS_DIR}/kirk.pem _output/admin-cert
+	cp ${ES_CERTS_DIR}/kirk-key.pem _output/admin-key
+	cp ${ES_CERTS_DIR}/root-ca.pem _output/admin-ca
+else
 	mkdir -p ${TLS_CERTS_BASEDIR}||:  && \
 	for n in "ca" "cert" "key" ; do \
-		oc -n openshift-logging get secret elasticsearch -o jsonpath={.data.admin-$$n} | base64 -d > _output/admin-$$n ; \
-	done && \
-	oc -n openshift-logging  get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name} > _output/espod && \
-	oc -n openshift-logging exec -c elasticsearch $$(cat _output/espod) -- cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt > _output/ca.crt && \
-	oc -n openshift-logging serviceaccounts get-token elasticsearch > _output/sa-token && \
-	echo openshift-logging > _output/namespace && \
-	mkdir -p /var/run/secrets/kubernetes.io/serviceaccount/||:  && \
-	sudo ln -sf $${PWD}/_output/ca.crt /var/run/secrets/kubernetes.io/serviceaccount/ca.crt && \
-	sudo ln -sf $${PWD}/_output/sa-token /var/run/secrets/kubernetes.io/serviceaccount/token && \
-	sudo ln -sf $${PWD}/_output/namespace /var/run/secrets/kubernetes.io/serviceaccount/namespace
-	
-.PHONY: prep-for-run
+		oc -n ${NAMESPACE} get secret elasticsearch -o jsonpath={.data.admin-$$n} | base64 -d > _output/admin-$$n ; \
```

Comment:
this can definitely and easily be replaced with `oc extract` outside of the for-loop:
```
oc -n ${NAMESPACE} extract secret/elasticsearch --to=_output/
```

Reply:
ok - then leave it as-is - we can clean up the code in a subsequent PR - let's just get this thing being used and tested asap

Reply:
#7